common/src/stack/command/stack/commands/__init__.py

```diff
@@ -1802,7 +1802,7 @@ def call(self, command, args=[]):
 	def notify(self, message):
 		print(f'{_logPrefix}{message}', file = sys.stderr, flush = True)
 
-	def command(self, command, args=[]):
+	def command(self, command, args=[], verbose_errors = True):
```

Comment:
Don't put spaces around the '=' in function parameters. It is wrong and you should be stoned for it.